### ISSUE TITLE
Explain purpose of esmorph

### ISSUE THREAD

**Issue by medikoo** [ariya/esmorph#6]
I really admire esprima, it's best parser around.

I found link to esmorf on esprima github home page, but neither there or here it's explained what really esmorph does.
"Source code modification" can be done just with esprima isn't it?

A simple example of usage would also be helpful

**Comment by ariya** [comment]
Indeed, the README will be populated with more info in the near future.

**Comment by goloroden** [comment]
Any news on this ;-)?

**Comment by popomore** [comment]
Is this project hold?

**Comment by iammerrick** [comment]
Bump.
